karma.conf.js

```diff
@@ -1,46 +1,16 @@
-// eslint-disable-next-line @typescript-eslint/no-var-requires
-const webpackConfig = require('./webpack.conf');
-
 module.exports = (config) => {
   config.set({
-    basePath: '',
-    frameworks: ['jasmine'],
+    plugins: ['karma-jasmine', 'karma-vite', 'karma-chrome-launcher', 'karma-sauce-launcher'],
+    frameworks: ['jasmine', 'vite'],
     files: [
-      'test/parchment.ts',
-      'test/setup.js',
-      'test/registry/*.js',
-      'test/unit/linked-list.js', // Control test order
-      'test/unit/registry.js',
-      'test/unit/attributor.js',
-      'test/unit/blot.js',
-      'test/unit/parent.js',
-      'test/unit/scroll.js',
-      'test/unit/container.js',
-      'test/unit/block.js',
-      'test/unit/inline.js',
-      'test/unit/embed.js',
-      'test/unit/text.js',
-      'test/unit/lifecycle.js',
-    ],
-    preprocessors: {
-      'test/registry/*.js': ['babel'],
-      'test/parchment.ts': ['webpack'],
-    },
-    mime: {
-      'text/x-typescript': ['ts'],
-    },
-    webpack: webpackConfig,
-    webpackMiddleware: {
-      stats: {
-        assets: false,
-        chunks: false,
-        colors: true,
-        errorDetails: true,
-        hash: false,
-        timings: false,
-        version: false,
+      {
+        // test ordering doesn't matter
```

Comment:
Let's remove the comment. We only need a comment if the order matters as that would be counterintuitive.

Reply:
fixed